pkgs/development/compilers/avian/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, zlib, openjdk }:
+
+stdenv.mkDerivation {
+  name = "avian-1.2.0";
+
+  src = fetchurl {
+    url = https://readytalk.github.io/avian-web/avian-1.2.0.tar.bz2;
```

Comment:
Why do they distribute the tarball on another repo?
I'd prefer fetchFromGitHub over fetchurl if it works but either should be fine